lib/capabilities.js

```diff
@@ -37,44 +36,17 @@ var capabilities = {
     checks: [
       {
         check: function(cb) {
-          if (license('agent:watchdog')) {
-            cb(true);
-          } else {
-            cb(false, 'Smart profiling license is missing or expired.');
-          }
-        }
-      },
-      {
-        check: function(cb) {
-          if (agent().internal.supports.watchdog) {
-            cb(true);
-          } else {
-            cb(false, 'Host platform does not support smart profiling.');
-          }
+          cb(false, 'Appmetrics does not support smart profiling.');
         }
       },
-      {
-        check: function(cb) {
-          var addon = require('strong-agent/lib/addon');
-          if (addon) {
-            cb(true);
-          } else {
-            cb(false, 'Strong-agent addon not compiled.');
-          }
-        }
-      }
     ]
   },
   tracing: {
     name: 'Tracing',
     checks: [
       {
         check: function(cb) {
-          if (license('agent:tracing')) {
-            cb(true);
-          } else {
-            cb(false, 'Tracing license is missing or expired.');
-          }
+          cb(false, 'Appmetrics does not support tracing.');
```

Comment:
no :-(. this "tracing" is what we called the strongops "slow endpoints". The other tracing is the strong-trace/concurix tracing.